### PR TITLE
set wpml language

### PR DESCRIPTION
**Pull request by TsankoBg**
https://basecamp.com/2523825/projects/17042950/todos/414457377

On checkout assign order wpml_language to the customer language - ICL_LANGUAGE_CODE

### CODE REVIEW

**Review by apeschar — CHANGES_REQUESTED** [review]
Check whether `ICL_LANGUAGE_CODE` is defined.

**Comment by TsankoBg** [comment]
I noticed that there is a ticket assigned to you about the plugin not being compatible with the newest version of WoocCommerce.  In my docker env, I also encountered this error as well while trying to install the plugin directly from the Shop, but it works fine when the plugin is installed from a zip.
https://basecamp.com/2523825/projects/17042950/todos/415155178

**Review by apeschar — CHANGES_REQUESTED** [review]
You didn't test this, did you?

~~~php
<?php
var_dump(!empty(YOLO));
~~~

~~~
% php test.php
PHP Warning:  Use of undefined constant YOLO - assumed 'YOLO' (this will throw an Error in a future version of PHP) in /home/albert/test.php on line 2
bool(true)
~~~

Every time you submit a PR without having tested it, God kills a kitten.

**Comment by apeschar** [comment]
> I noticed that there is a ticket assigned to you about the plugin not being compatible with the newest version of WoocCommerce. In my docker env, I also encountered this error as well while trying to install the plugin directly from the Shop, but it works fine when the plugin is installed from a zip.
> https://basecamp.com/2523825/projects/17042950/todos/415155178

Thanks. Could you update the `WC tested up to:` in webwinkelkeur/webwinkelkeur.php to `4.99`, and try again?  I think that should fix the issue.

**Comment by apeschar** [comment]
https://www.php.net/defined

But please, stop just assuming shit works.  Test it.

**Comment by TsankoBg** [comment]
I messed up the tests....

**Comment by TsankoBg** [comment]
> Thanks. Could you update the WC tested up to: in webwinkelkeur/webwinkelkeur.php to 4.99, and try again? I think that should fix the issue.

Installing it from a zip file does not show any warnings.

File: common/woocommerce.php [hunk 1]
```diff
@@ -6,6 +6,15 @@ class WebwinkelKeurWooCommerce extends WebwinkelKeurCommon {
     public function __construct(array $settings) {
         parent::__construct($settings);
         add_action('woocommerce_order_status_completed', array($this, 'order_completed'), 10, 1);
+        add_action('woocommerce_checkout_update_order_meta', [$this, 'set_order_language']);
+    }
+
+    public function set_order_language($order_id) {
+        if (!get_post_meta($order_id, 'wpml_language')) {
+            if (defined('ICL_LANGUAGE_CODE')) {
```
Comment (apeschar) [inline common/woocommerce.php]: What happened to `&&` ? :)

**Review by apeschar — CHANGES_REQUESTED** [review]

File: bin/gen_readme [hunk 2]
```diff
@@ -27,7 +27,7 @@ Contributors: apeschar
 Tags: {$info['readme']['tags']}
 Requires at least: 4.4
 Requires PHP: 5.6
-Tested up to: 5.2
+Tested up to: 5.4
```
Comment (TsankoBg) [inline bin/gen_readme:30]: It shows a warning in the WordPress store

**Review by apeschar — APPROVED** [review]

**Pull request merged**